tests/test_marginals.py

```diff
@@ -0,0 +1,55 @@
+import numpy as np
+import pandas as pd
+from scipy.stats import norm
+import unittest
+
+from context import grama as gr
+from context import models, data
+
+## Marginals function tests
+##################################################
+class TestMarginals(unittest.TestCase):
```

Comment:
Consolidating all marginal tests makes it clear that these two classes `TestMarginals` and `TestMarginal` are poorly named. Suggest changing `TestMarginals` to `TestMarginalTools` to better reflect the test content.